content/millennium/r4/care-provision/service-request.md

```diff
@@ -0,0 +1,122 @@
+---
+title: ServiceRequest | R4 API
+---
+
+# ServiceRequest
+
+* TOC
+{:toc}
+
+## Overview
+
+ServiceRequest is a record of a request for a Service or diagnostic or other service to be planned, proposed, or performed, as distinguished by the ServiceRequest.intent field value, with or on a patient. The Service will lead to either a Service or DiagnosticReport, which in turn may reference one or more Observations, which summarize the performance of the Services and associated documentation such as observations, images, findings that are relevant to the treatment/management of the subject. This resource may be used to share relevant information required to support a referral or a transfer of care request from one practitioner or organization to another when a patient is required to be referred to another provider for a consultation /second opinion and/or for short term or longer term management of one or more health issues or problems.
+
+The principal intention of ServiceRequest is to support ordering Services for one patient (which includes non-human patients in veterinary medicine). However, in many contexts, healthcare related processes include performing diagnostic investigations on groups of subjects, devices involved in the provision of healthcare, and even environmental locations such as ducts, bodies of water, etc. ServiceRequest supports all these usages. The service request may represent an order that is entered by a practitioner in a CPOE system as well as a proposal made by a clinical decision support (CDS) system based on a patient's clinical record and context of care. Planned Services referenced by a CarePlan may also be represented by this resource.
+
+
+
+The following fields are returned if valued:
+
+* [ServiceRequest id]( https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.status){:target="_blank"}
+* [Intent](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.intent){:target="_blank"}
+* [Category](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.category){:target="_blank"}
+* [Priority](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.priority){:target="_blank"}
+* [Service](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.code){:target="_blank"}
+* [Order Details](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.orderDetail){:target="_blank"}
+* [Quantity](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.quantity_x_){:target="_blank"}
+* [Patient](https://www.hl7.org/fhir/servicerequest-definitions.html#ServiceRequest.subject){:target="_blank"}
```

Comment:
[Patient] seems reasonable, but should it be 'Subject' to agree with attribute

Reply:
After speaking with the solution designer, it was decided to keep it Patient